dotnet/src/Plugins/Plugins.Core/CodeInterpreter/SessionPythonSettings.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.ComponentModel;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Plugins.Core.CodeInterpreter;
+
+/// <summary>
+/// Settings for a Python session.
+/// </summary>
+public class SessionPythonSettings
```

Comment:
I see Endpoint is nullable, but I would expect it to be a required setting for the service to work.  is it like this because we have one class for both public settings and what we are passing internally to the service?  If so, should we split it, so that it's more obvious to users what settings they have to provide at programming time, rather than run time?

Reply:
Moved to be a required item.

Splitted the Model for request from the Plugin settings, to give a better idea of what is a setting in the plugin level.